tests/test_chi_ssa_27.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import COMMISSION, PASSED, TENTATIVE
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.chi_ssa_27 import ChiSsa27Spider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "chi_ssa_27.html"),
+    url="https://www.lakeviewchamber.com/ssa27",
+)
+spider = ChiSsa27Spider()
+
+freezer = freeze_time("2019-07-03")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
```

Comment:
It's less relevant now, but once you add committees it'll be good to add a `test_count` function to make sure that the right meetings are being included like we have here https://github.com/City-Bureau/city-scrapers/blob/bc1d5ff6163517df4e198dcd6161754a0a326d7c/tests/test_cook_board.py#L17

Reply:
Ok will do.